package.json

```diff
@@ -366,6 +366,11 @@
           "type": "boolean",
           "default": true,
           "description": "Specifes whether OmniSharp should use VS Code editor settings for C# code formatting (use of tabs, indentation size)."
+        },
+        "omnisharp.format.enable": {
```

Comment:
It's at the "omnisharp" namespace because it ends up affecting whether particular flags are passed to OmniSharp on launch. With the advantage of hindsight, I don't think "omnisharp.useEditorFormattingSettings" should have been added. This should have been automatic. @filipw, do you agree?